server.js

```diff
@@ -25,8 +25,14 @@ app.use(express.static('public'));
 app.engine('handlebars', exphbs({
   defaultLayout: 'main',
   helpers: {
+    upperCase: function(str) {
```

Comment:
@catchbb14  typically converting styles would be done within css. however this does technically still work. just a note, that in a real world situation, it'd be more commonly known that this would be handled within css. approving anyway because this project is for funzies/learnings. nice experiment i wouldn't have thought of using a helper in a way like this.